website/src/content/help-needed.mdx

```diff
@@ -125,3 +125,15 @@ export const entryPointUriPath = typeof window === 'undefined'
 export const PERMISSIONS = entryPointUriPathToPermissionKeys(entryPointUriPath);
 ```
 </Info>
+
+
+# Current Limitations
+
+As we continue developing Custom Applications further introducing more and more developer-friendly features, our solution still has some limitations. Please find below the list of things we don’t support at the moment:
+
+* Changing OAuth Scopes for a Custom Application. The changes to permission scopes are not allowed in the “Ready” state for Custom Applications. At the moment, to apply such changes, a user needs to change the state of a Custom Application to “Draft”, update permissions, and then move it back to the “Ready” state to continue working with the application.
+* With a new project created, permissions for already existing Custom Applications are not updated. The current workaround involves reinstalling the application and installing again for all required projects.
+* Only View/Manage permissions are available for Custom Applications at the moment. Decoupling from the internal permission scopes was an important part of the migration of Custom Applications to the Organizaiton level. This allows us to continue developing the functionality without the project-level limitations.
+
+Please [reach out to us](https://github.com/commercetools/merchant-center-application-kit/discussions) in case you found any workaround for the aforementioned limitations and constraints. We are always happy to hear your ideas and general feedback about using Custom Applications.
```

Comment:
```suggestion
Some of the features recently released have some known limitations that we plan to address as a follow up.

_Last update 2022-06-26_

## Changing OAuth Scopes for a Custom Application in Ready state

When you [configure a Custom Application in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications#configuring-custom-applications) you can change the list of OAuth Scopes assigned to the Custom Application permissions as long as the Custom Application is in **Draft** state.

Once the Custom Application is in **Ready** state, the list of OAuth Scopes cannot be changed anymore because the Custom Application can be installed and used while in that state.

Changing OAuth Scopes requires to [move the Custom Application back to **Draft** state](https://docs.commercetools.com/merchant-center/managing-custom-applications#moving-the-custom-applications-to-the-draft-state), update the list of OAuth Scopes, change the state to **Ready** again and go through the installation process again.

<Info>

Changing state from **Ready** to **Draft** results in the Custom Application being uninstalled from all Organizations and Projects. This means that [all Team permissions must be assigned again](https://docs.commercetools.com/merchant-center/managing-custom-applications#assigning-team-permissions) once the Custom Application is being installed again.

</Info>

## Project access is not automatically updated for newly created Projects

For an installed Custom Application, if Project access has been configured to **Install for all projects of this organization**, when a new Project is created the access to that Project is not automatically assigned.

To do so, you need to manually [update Project access](https://docs.commercetools.com/merchant-center/managing-custom-applications#updating-projects-access) for the installed Custom Application.

## Custom Applications have only one set of permissions

Currently each Custom Application has a [unique pair of user permissions](/concepts/oauth-scopes-and-user-permissions) assigned to them.

This change and restriction was an important step in being able to release Org-level Custom Application. You can [read more about the reasoning in the migration document](/migrating-from-project-level-custom-applications#oauth-scopes-and-user-permissions).

However, in the future we will allow more than one pair of user permissions to be defined for each Custom Application.
```